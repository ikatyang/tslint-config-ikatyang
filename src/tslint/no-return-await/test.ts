// good

async function good1() {
  return Promise.resolve(null);
}

// bad

async function bad1() {
  return await Promise.resolve(null);
}
