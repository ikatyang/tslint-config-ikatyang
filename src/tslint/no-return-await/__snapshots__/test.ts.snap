// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

async function good1() {
  return Promise.resolve(null);
}

// bad

async function bad1() {
  return await Promise.resolve(null);
         ~~~~~                        [Unnecessary 'await'.]
}

`;
